data-raw/healthy-people/happiness.R

```diff
@@ -1,3 +1,31 @@
+# ---- Load packages ----
+library(tidyverse)
+library(rio)
+
+# ---- Get and clean data ----
+# Happiness
+# Source: https://www.ons.gov.uk/datasets/wellbeing-local-authority/editions/time-series/versions/4
+
+happiness_raw <- import("https://download.ons.gov.uk/downloads/datasets/wellbeing-local-authority/editions/time-series/versions/4.csv")
+
+people_happiness <- happiness_raw |>
+  filter(
+    str_starts(`administrative-geography`, "N"),
+    MeasureOfWellbeing == "Happiness",
+    `yyyy-yy` == "2022-23",
+    `wellbeing-estimate` == "average-mean"
+  ) |>
+  slice(-11) |>
+  select(
+    ltla24_code = `administrative-geography`,
+    happiness_score_out_of_10 = `v4_3`,
+    year = `Time`
+  )
+
+# ---- Save output to data/ folder ----
+usethis::use_data(people_happiness, overwrite = TRUE)
+
+
 # ---- Load ----
```

Comment:
I'm confused as to what is going on here. It looks like there are two analyses in the same file. I presume one is the old analysis? If so, please can you delete the analyses you don't want and resubmit?

Reply:
deleted and resubmitted